test/auto_parallel/spmd_rules/test_embedding_rule.py

```diff
@@ -110,6 +113,66 @@ def test_embedding_infer_forward(self):
                 self.attrs,
             )
 
+    def test_embedding_infer_backward(self):
+        # backward setup
+        out_shape = [4, 1024, 768]  # [B,S, H]
+        process_mesh = auto.ProcessMesh(mesh=[[0, 1, 2, 3], [4, 5, 6, 7]])
+
+        out_tensor_dist_attr = TensorDistAttr()
+        out_tensor_dist_attr.process_mesh = process_mesh
+        self.out_dist_tensor_spec = DistTensorSpec(
+            out_shape, out_tensor_dist_attr
+        )
+
+        self.attrs = {
+            'padding_idx': -1,
+            'sparse': False,
+        }
+
+        # data parallel
+        self.out_dist_tensor_spec.set_dims_mapping([1, -1, -1])
+        result_dist_attrs = self.rule1.infer_backward(
+            [self.out_dist_tensor_spec], self.attrs
```

Comment:
the inputs of infer_backward includes input_specs and output_specs

Reply:
fixed